workspaces/libnpmexec/test/fixtures/setup.js

```diff
@@ -118,35 +119,19 @@ const createTestdir = (...objs) => {
   return testdirHelper(merge(...objs))
 }
 
-const mockNpxInventoryCache = (packageName, fakeVersion) => {
-  const Arborist = require('@npmcli/arborist')
-  const originalLoadActual = Arborist.prototype.loadActual
-
-  Arborist.prototype.loadActual = async function () {
-    const tree = await originalLoadActual.call(this)
-    if (this.path.includes('npxCache')) {
-      const originalQuery = tree.inventory.query.bind(tree.inventory)
-      tree.inventory.query = (key, name) => {
-        if (key === 'packageName' && name === packageName) {
-          return [{
-            package: {
-              version: fakeVersion,
-              name: packageName,
-              resolved: 'dummy-resolved',
-            },
-            pkgid: `${packageName}@${fakeVersion}`,
-            depth: 0,
-          }]
-        }
-        return originalQuery(key, name)
-      }
-    }
-    return tree
-  }
-
-  return () => {
-    Arborist.prototype.loadActual = originalLoadActual
+const setupNpxInventory = async (basePath, name, version, packageSpec) => {
```

Comment:
This really should be the domain of `setup` in the test itself, we're already passing the directory contents to the main `setup` directory.